build.sh

```diff
@@ -19,11 +22,13 @@ cd "${pwdir}"
 pwdir="`pwd`"
 
 system () {
-	install -o root -g wheel -m 755 -d "${root}"
+	mkdir -p "${src}" || return 1
+	mkdir -p "${root}" || return 1
```

Comment:
The use of `install` command ensures the root is created with correct user, group and mode, so please don't replace it with a simple `mkdir -p`.